ratpack-codahale-metrics/src/main/java/ratpack/codahale/metrics/CodaHaleMetricsModule.java

```diff
@@ -651,6 +680,81 @@ public Csv reportDirectory(File reportDirectory) {
       }
     }
 
+    public static class Graphite {
```

Comment:
I can relocate these "builder" classes to another package.
What do you say about "ratpack.codahale.metrics.internal.builder" or "ratpack.codahale.metrics.builder" ?